apps/wallet/src/ui/app/experimentation/features.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * This is a list of feature keys that are used in
+ * in https://docs.growthbook.io/app/features#feature-keys
+ */
+export enum FEATURES {
+    DEPRECATE_GATEWAY = 'deprecate-gateway',
+    RPC_API_VERSION = 'rpc-api-version',
```

Comment:
Looks like this isn't feature gated

Reply:
Can you elaborate on this? This is used in https://github.com/MystenLabs/sui/blob/87e1314ef61fc39904a612bcf9d96481065f02bb/apps/wallet/src/ui/app/ApiProvider.ts#L82